v2/robotmk/runner.py

```diff
@@ -58,15 +58,18 @@ def command(self) -> list[str]:
         if self.argument_file is not None:
             robot_command.append(f"--argumentfile={self.argument_file}")
         if self.retry_strategy is RetryStrategy.INCREMENTAL and self.index > 0:
-            robot_command.append(f"--rerunfailed={self._output_file(self.index - 1)}")
+            robot_command.append(
+                f"--rerunfailed={self._output_file(self.index - 1, 'xml')}"
+            )
         return robot_command + [
             f"--outputdir={self.output_directory}",
-            f"--output={self.output_file()}",
+            f"--output={self.output_xml_file()}",
+            f"--log={self._output_file(self.index, 'html')}",
             str(self.robot_target),
         ]
 
-    def _output_file(self, index: int) -> pathlib.Path:
-        return self.output_directory.joinpath(f"{index}.xml")
+    def _output_file(self, index: int, suffix: Literal["xml", "html"]) -> pathlib.Path:
```

Comment:
This seems like one indirection more than needed, and the signature is mixing `self.index` and `index`. Make this a classmethod? But I prefer removing the method.

Reply:
Done